utils/index.ts

```diff
@@ -1,9 +1 @@
-// @ts-nocheck
-export function shuffleArray(array) {
-  return (
-    array
-      ?.map((a) => ({ sort: Math.random(), value: a }))
-      .sort((a, b) => a.sort - b.sort)
-      .map((a) => a.value) || []
-  );
-}
+export { getSpeakerSocialsLinks } from "./getSpeakerSocialsLinks";
```

Comment:
```suggestion
export * as default from "./getSpeakerSocialsLinks";
```

Pour éviter d'avoir des imports dupliqués par Nuxt